recipes/r-matchit/LICENCE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Please remove that file.. it is already included into the package via the `'{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'` reference.